frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -224,6 +224,13 @@ function Table(props: ResourceTableProps) {
                 return 0;
               },
             };
+          case 'categories':
```

Comment:
Is categories common enough in k8s resources that it would save us time having a predefined column? (I think it is not, but I may be mistaken)